### PR TITLE
[SMALLFIX] Make spot instance request support fault tolerant Tachyon

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

File: deploy/vagrant/create [hunk 1]
```diff
@@ -181,7 +181,7 @@ if [[ "$TACHYON_MASTERS" -gt 1 ]]; then
   for master in ${masters[@]}; do
     info "Tachyon master: ${master}"
     info "Trying to curl Tachyon web UI at ${master}:19999"
-    curl "${master}:19999" >/dev/null
+    curl "${master}:19999" >/dev/null || true # Ignore error of `curl`
```
Comment (uronce-cc) [inline deploy/vagrant/create]: Oh, not right.

**Comment by uronce-cc** [comment]
@gpang PTAL

**Comment by gpang** [comment]
This works on my machine. Thanks @uronce-cc !

**Comment by gpang** [comment]
LGTM

**Pull request merged**